frontend/src/utils/utils.ts

```diff
@@ -30,18 +30,43 @@ export const round = (value: number | string, decimals = 2) => {
 
 /**
  * @notice Rounds to appropriate human readable decimals for the token
- * @dev This is very fragile and should be replace with something robust, i.e. something that walks
- * value and rounds considering sig-figs or similar
+ * @param amount
+ * @param token
```

Comment:
We should probably either add some words describing these inputs, or remove these two lines (I'm fine with either). As-is they just kinda take up space (since we'll never autogenerate docs for the frontend anyway, i.e. the NatSpec comments here are just for devs, not generated docs)

Reply:
Did know this, thanks